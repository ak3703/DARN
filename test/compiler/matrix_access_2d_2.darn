int main() { 
	int[5][5] m;

	int i;
	int j;

	for (i=0; i<5;i=i+1) {
		for (j=0; j<5;j=j+1) {
				m[i][j] = i+j;			
		}
	}

	for (i=0; i<5;i=i+1) {
		for (j=0; j<5;j=j+1) {
			print(m[i][j]);
		}
	}	

}
