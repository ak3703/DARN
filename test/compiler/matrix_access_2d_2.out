0123412345234563456745678